model/src/main/java/org/projectnessie/api/params/AbstractParams.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.api.params;
+
+import javax.ws.rs.QueryParam;
+import org.eclipse.microprofile.openapi.annotations.parameters.Parameter;
+
+public abstract class AbstractParams {
+
+  @Parameter(description = "maximum number of entries to return, just a hint for the server")
+  @QueryParam("max")
+  private Integer maxRecords;
```

Comment:
`maxRecordsPerPage` perhaps for the sake of clarity?

Reply:
Let's not change unrelated things in this PR.

Reply:
ok